dns-nas-utils/etc/dns-nas-utils.conf

```diff
@@ -0,0 +1,18 @@
+#Use uncommented lines to change daemon configuration.
+
+# High frequency sleep time (low frequency is high*10)
+#frequency=0.75
+
+# Fan hysteresis, low and high thresholds.
+#low_thresh=45
+#high_thresh=50
+#hysteresis=2
+
+# Temporary path for some file exchanges between different calls.
+#tmppath="/run/dns-nas-utils"
+
+# Default fan device
+#fan_device="/sys/devices/gpio_fan/hwmon/hwmon0/fan1_target"
```

Comment:
:+1: My DNS has different path to the fan. It is great that I can configure it from here!